controllers/storage/provision.go

```diff
@@ -209,22 +249,30 @@ func (m *MinIOReconciler) generateSwiftSecret() *corev1.Secret {
 		},
 		Type: corev1.SecretTypeOpaque,
 		Data: map[string][]byte{
-			"accesskeyid":     []byte(m.HarborCluster.Spec.Storage.Oss.Accesskeyid),
-			"accesskeysecret": []byte(m.HarborCluster.Spec.Storage.Oss.Accesskeysecret),
-			"region":          []byte(m.HarborCluster.Spec.Storage.Oss.Region),
-			"bucket":          []byte(m.HarborCluster.Spec.Storage.Oss.Bucket),
-			"endpoint":        []byte(m.HarborCluster.Spec.Storage.Oss.Endpoint),
-			"internal":        []byte(m.HarborCluster.Spec.Storage.Oss.Internal),
-			"encrypt":         []byte(m.HarborCluster.Spec.Storage.Oss.Encrypt),
-			"secure":          []byte(m.HarborCluster.Spec.Storage.Oss.Secure),
-			"rootdirectory":   []byte(m.HarborCluster.Spec.Storage.Oss.Rootdirectory),
-			"chunksize":       []byte(m.HarborCluster.Spec.Storage.Oss.Chunksize)},
+			"authurl":             []byte(m.HarborCluster.Spec.Storage.Swift.Authurl),
```

Comment:
There are many fields didn't match `CamelCase`. Such as
`Authurl`, `Username`, `Tenantid`...

Reply:
all params are lowercase in secret, ref [here](https://github.com/goharbor/harbor-operator/blob/c8e922bded7080598ee5d10861271b792a22086d/docs/installation/registry-storage-configuration.md#create-the-secret)